eng/pipelines/templates/steps/workload-publish.yml

```diff
@@ -0,0 +1,28 @@
+parameters:
+- name: publishToFeed
+  displayName: Publish to feed
+  type: boolean
+  default: false
+- name: feedForPublishing
+  displayName: Feed for publishing
+  type: string
+  default: public/dotnet10-workloads
+
+steps:
+- script: |
+    echo "publishToFeed: ${{ parameters.publishToFeed }}"
+    echo "feedForPublishing: ${{ parameters.feedForPublishing }}"
+    echo "Build.SourcesDirectory: $(Build.SourcesDirectory)"
+  displayName: Print Parameters
+  condition: ${{ eq(parameters.publishToFeed, true) }}
+# - task: 1ES.PublishNuget@1
```

Comment:
Should these be removed since they're commented out?

Reply:
No, the idea here is to use the `1ES.PublushNuget@1` task, but left that commented out while testing

Reply:
Please don't use 1ES.PublushNuget@1. That task doesn't exist.